packages/manager/modules/anthos/src/dashboard/routing.js

```diff
@@ -13,7 +14,9 @@ export default /* @ngInject */ ($stateProvider) => {
       alertId: () => 'anthos_dashboard',
 
       tenant: /* @ngInject */ (serviceName, AnthosTenantsService) => {
-        return AnthosTenantsService.getTenantDetails(serviceName);
+        return AnthosTenantsService.getTenantDetails(serviceName).then(
```

Comment:
**nitpick**
You could use return shortcut (`=> AnthosTenantsService.getTenantDetails(…) ...`)